Gemfile

```diff
@@ -23,7 +23,9 @@ gem 'puma', '~> 5.0'
 # gem 'redis', '~> 4.0'
 # Use Active Model has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
-
+gem 'jsonapi.rb'
+gem 'jsonapi-serializer'
```

Comment:
It's a big chance that I misled you. We don't have to add `jsonapi-serializer'` cause it's already included in `jsonapi.rb` gem. Also remember to add version to `jsonapi.rb` gem